examples/renderhint/src/index.js

```diff
@@ -0,0 +1,160 @@
+        : stats[0].remoteVideoTrackStats[0]
+      const bytesReceived = remoteTrackStats.bytesReceived;
+      const timestamp = remoteTrackStats.timestamp;
+      const bitrate = Math.round((bytesReceivedPrev - bytesReceived) * 8 / (timestampPrev - timestamp));
+
+      bitrateSeries.addPoint(timestamp, bitrate);
+      bitrateGraph.setDataSeries([bitrateSeries]);
+      bitrateGraph.updateEndDate();
+      bytesReceivedPrev = bytesReceived;
+      timestampPrev = timestamp;
+    }, intervalMs);
+
+    bitrateGraph.graphDiv_.style.display = '';
+    return function stop() {
+      clearInterval(interval);
+      bitrateGraph.graphDiv_.style.display = 'none';
+    };
+  };
+}
+
+const handleIsSwitchedOff = (trackState) => {
+  if(trackState) {
+    trackIsSwitchedOff.textContent = 'Off';
+    trackIsSwitchedOff.classList.remove('badge-success');
+    trackIsSwitchedOff.classList.add('badge-danger');
+  } else {
+    trackIsSwitchedOff.textContent = 'On';
+    trackIsSwitchedOff.classList.remove('badge-danger');
+    trackIsSwitchedOff.classList.add('badge-success');
+  }
+}
+
+(async function(){
+  // Load the code snippet.
+  const snippet = await getSnippet('./helpers.js');
+  const pre = document.querySelector('pre.language-javascript');
+
+  pre.innerHTML = Prism.highlight(snippet, Prism.languages.javascript);
+
+  const logger = Video.Logger.getLogger('twilio-video');
+  logger.setLevel('silent');
+
+  // Get the credentials to connect to the Room.
+  const credsP1 = await getRoomCredentials();
+  const credsP2 = await getRoomCredentials();
+
+  // Create room instance and name for participants to join.
+  roomP1 = await joinRoom(credsP1.token);
+
+  // Create the video track for the Remote Participant.
+  const videoTrack = await Video.createLocalVideoTrack();
+
+  // Connecting remote participant.
+  const roomP2 = await Video.connect(credsP2.token, {
+    name: 'my-cool-room',
+    bandwidthProfile: {
+      video: {
+        contentPreferencesMode: 'manual',
+        clientTrackSwitchOffControl: 'manual'
+      }
+    },
+    tracks: [ videoTrack ]
+  });
+
+  // Set video bitrate graph.
+  startVideoBitrateGraph = setupBitrateGraph('video', 'videobitrategraph', 'videobitratecanvas');
+
+  // Attach RemoteVideoTrack
+  roomP1.on('trackSubscribed', track => {
+    if(track.kind === 'video') {
+      track.attach(videoEl);
+      remoteVideoTrack = track;
+      handleIsSwitchedOff(track.isSwitchedOff);
+      stopVideoBitrateGraph = startVideoBitrateGraph(roomP1, 1000);
+
+      switchOnOffBtn.classList.remove('disabled');
+      renderDimensionsOption.classList.remove('disabled');
+
+      remoteVideoTrack.on('switchedOff', track => {
+        handleIsSwitchedOff(track.isSwitchedOff);
+      });
+      remoteVideoTrack.on('switchedOn', track => {
+        handleIsSwitchedOff(track.isSwitchedOff);
+      });
+    }
+  });
+
+  // Remote Track Switch On/Off
+  switchOnOffBtn.onclick = event => {
```

Comment:
@PikaJoyce, I preferred the on and off being separate buttons. That way it's clear that track switchOn can be called even when track is switchedOn and such.

also with them being same button now its possible that the text of the button can be misleading from what button does. For example  you may have button text saying "Switch On", but track could be in switched Off state (bandwidth profile algorithm can switch off tracks if bandwidth is limited). and since the button action checks for `remoteVideoTrack.isSwitchedOff` it will endup calling switchOn instead of switchOff.

Reply:
Thanks for the suggestion @makarandp0. I'll just swap it back to the separate buttons for now :)! @manjeshbhargav seemed to like the idea of a singular/toggle button instead so I'll ping him here in case for future references. 

I'll separate the buttons and functionality again then!